pkgs/games/openmw/default.nix

```diff
@@ -99,35 +103,34 @@ stdenv.mkDerivation rec {
     SDL2
     boost
     bullet'
-    ffmpeg_6
+    ffmpeg
     libXt
     luajit
     lz4
     mygui
     openal
     osg'
+    qttools
     recastnavigation
     unshield
     yaml-cpp
   ];
 
-  cmakeFlags =
-    [
-      "-DOpenGL_GL_PREFERENCE=${GL}"
-      "-DOPENMW_USE_SYSTEM_RECASTNAVIGATION=1"
-    ]
-    ++ lib.optionals stdenv.hostPlatform.isDarwin [
-      "-DOPENMW_OSX_DEPLOYMENT=ON"
-    ];
+  cmakeFlags = [
+    (lib.cmakeFeature "OpenGL_GL_PREFERENCE" withGL)
+    (lib.cmakeBool "OPENMW_USE_SYSTEM_RECASTNAVIGATION" true)
+  ] ++ lib.optional stdenv.hostPlatform.isDarwin (lib.cmakeBool "OPENMW_OSX_DEPLOYMENT" true);
```

Comment:
I guess you could just do `(lib.cmakeBool "OPENMW_OSX_DEPLOYMENT" stdenv.hostPlatform.isDarwin)` and drop the optional bit. It reads better.